jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalrun" -c "base ocnanal ocnanalrun"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
```

Comment:
```suggestion
```
This cannot be `gfs`, as the script is explicitly a `JGDAS` script.
Besides, both, `RUN` and `CDUMP` are defined upstream (has to), so this line accomplishes nothing.